DuckDuckGo/NavigationBar/View/NavigationBarViewController.swift

```diff
@@ -419,8 +419,13 @@ final class NavigationBarViewController: NSViewController {
     }
 
     func listenToFeedbackFormNotifications() {
-        feedbackFormCancellable = NotificationCenter.default.publisher(for: .OpenUnifiedFeedbackForm).receive(on: DispatchQueue.main).sink { _ in
-            WindowControllersManager.shared.showShareFeedbackModal(source: .ppro)
+        feedbackFormCancellable = NotificationCenter.default.publisher(for: .OpenUnifiedFeedbackForm).receive(on: DispatchQueue.main).sink { notification in
```

Comment:
💡 Follow what I’ve added above. We can leverage the current enum to encapsulate this logic in one place:

```swift
func listenToFeedbackFormNotifications() {
    feedbackFormCancellable = NotificationCenter.default
        .publisher(for: .OpenUnifiedFeedbackForm)
        .receive(on: DispatchQueue.main)
        .sink { notification in
            let source = UnifiedFeedbackSource.source(from: notification.userInfo)
            WindowControllersManager.shared.showShareFeedbackModal(source: source)
        }
}
```

and also:

```swift
enum UnifiedFeedbackSource: String, StringRepresentable {
    private static let sourceKey = "source"

    case settings, ppro, vpn, pir, itr, unknown
    static var `default` = UnifiedFeedbackSource.unknown

    static func userInfo(source: UnifiedFeedbackSource) -> [String: Any] {
        return [sourceKey: source.rawValue]
    }

    static func source(from userInfo: [AnyHashable : Any]?) -> UnifiedFeedbackSource {
        if let userInfo = userInfo as? [String: Any], let source = userInfo[sourceKey] as? String {
            return UnifiedFeedbackSource(rawValue: source) ?? .ppro
        } else {
            return .ppro
        }
    }
}
```